polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -761,6 +767,58 @@ impl orml_xcm::Config for Runtime {
 	type SovereignOrigin = EnsureRoot<AccountId>;
 }
 
+parameter_type_with_key! {
+	pub ExistentialDeposits: |currency_id: CurrencyId| -> Balance {
+		match currency_id {
+			CurrencyId::TEER => MILLITEER,
+			CurrencyId::RelayNative => 3_333_333u128.into() // keep this with asset hub ED
```

Comment:
would be nicer to have KSM, but as we want to have one runtime for KSM and DOT, we may keep it generic